service/src/lib.rs

```diff
@@ -804,6 +819,39 @@ pub fn westend_new_full(
 	Ok((service, client, handles))
 }
 
+/// Create a new Westend service for a full node.
```

Comment:
```suggestion
/// Create a new Rococo service for a full node.
```

ironically the same mistake as fixed above : )